Src/BootCamp.Chapter/ConsoleArgumentsParser.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.IO;
+
+namespace BootCamp.Chapter
+{
+    public static class ConsoleArgumentsParser
+    {
+        public static void ArgumentsParser(string[] args)
```

Comment:
Raw code. This function is completely unorganized and should be split into private step functions. I don't want to read 60 lines of code to understand what it does.